common/static/common/js/components/views/paging_footer.js

```diff
@@ -17,23 +17,21 @@
                     this.collection.bind('add', _.bind(this.render, this));
                     this.collection.bind('remove', _.bind(this.render, this));
                     this.collection.bind('reset', _.bind(this.render, this));
-                    this.render();
```

Comment:
FYI I'm strongly in favor of not rendering in initialize method because it makes it hard to control the order of rendering.I also find that it messes up with subclassing because the view will be rendered before the subclass initializes (not that we subclass views very often).  So I'm in favor of this change as long as we are rendering from the parent's render method.